packages/manager/apps/pci-vouchers/package.json

```diff
@@ -55,10 +51,18 @@
     "eslint": "^8.56.0",
     "postcss": "^8.4.32",
     "rollup": "^4.4.0",
-    "tailwindcss": "^3.4.0",
-    "typescript": "^5.3.3",
-    "vite": "^3.0.2",
-    "vitest": "^1.2.0"
+    "tailwindcss": "^3.4.0"
+  },
+  "peerDependencies": {
+    "date-fns": "workspace:*",
+    "i18next": "workspace:*",
```

Comment:
why put those dependencies in peerDeps ?

a micro app is providing it's own react version, what if we have 30 or more µapp ? we'll have to bump all the apps at the same ; same issue for other packages added in peer dep